EMML/1-1000/EMML50.xml

```diff
@@ -324,6 +324,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                                  <ptr target="bm:EMML1"/>
```

Comment:
to line 294 above - I think the `placeName ref` to `LOC4506Manbar `should be replaced with `INS0101MadhaneAlam`

Reply:
I don't know. Anais, who knows the manuscript better, specifically wrote "in Gondar", which would be LOC4506Manbar. On the other hand, the marginal note does indeed look like the typical one from Maqdala.